src/VisualStudio/Core/Def/Implementation/ProjectSystem/FileChangeTracker.cs

```diff
@@ -87,11 +88,18 @@ public void StartFileChangeListeningAsync()
 
             Contract.ThrowIfTrue(_fileChangeCookie != s_none);
 
-            _fileChangeCookie = new Lazy<uint>(() =>
+            _fileChangeCookie = new Lazy<uint?>(() =>
```

Comment:
Should we detect cases when `EnsureSubscription` is called, but `StartFileChangeListeningAsync` has never been called.